### PR TITLE
Specify export flag for Context-based receiver

### PR DESCRIPTION
**Pull request by jmatsu**
This feature has been introduced since Android 13 SDK (https://developer.android.com/about/versions/13/features#runtime-receivers) and it's now required for apps that target Android 14 or later (https://developer.android.com/about/versions/14/behavior-changes-14#runtime-receivers-exported).

See the implementation of ContextCompat for the verification. https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:core/core/src/main/java/androidx/core/content/ContextCompat.java;drc=b63ff8c4299a6486bf34ec2ac746517e2d378d43

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,10 +3,10 @@ version: 2.1
 executors:
   android:
     docker:
-      - image: cimg/android:2022.03
+      - image: cimg/android:2023.07
     working_directory: ~/deploygate-android-sdk
     environment:
-      GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError" -Dorg.gradle.daemon=false'
+      GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xmx2048m -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemoryError" -Dorg.gradle.daemon=false'
```
Comment (jmatsu) [inline .circleci/config.yml:9]: MaxPermSize is no longer valid for JVM 9 or later

File: build.gradle [hunk 2]
```diff
@@ -7,10 +7,11 @@ buildscript {
     repositories {
         mavenCentral()
         google()
+        gradlePluginPortal()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.2'
-        classpath 'com.akaita.android:easylauncher:1.3.1'
+        classpath 'com.android.tools.build:gradle:7.4.2'
```
Comment (jmatsu) [inline build.gradle:13]: To use Android SDK 33

File: sdk.build.gradle [hunk 3]
```diff
@@ -40,11 +40,11 @@ android {
...
-    testImplementation 'androidx.test:runner:1.2.0'
-    testImplementation 'androidx.test.ext:junit:1.1.1'
-    testImplementation 'org.robolectric:robolectric:4.3.1'
+    testImplementation 'androidx.test:runner:1.5.2'
+    testImplementation 'androidx.test.ext:junit:1.1.5'
+    testImplementation 'org.robolectric:robolectric:4.10.3'
 
-    testImplementation 'androidx.test:rules:1.2.0'
+    testImplementation 'androidx.test:rules:1.5.0'
     testImplementation 'com.google.truth:truth:1.0'
```
Comment (satsukies) [inline sdk.build.gradle:48]: truth:1.1.4 is now available. how about it updating?
see: https://truth.dev/

```suggestion
    testImplementation 'com.google.truth:truth:1.1.4'
```

Comment (jmatsu) [reply sdk.build.gradle:48]: @satsukies 
Please make a new PR. These updates are just for Android 14 SDK and JVM changes.

Comment (satsukies) [reply sdk.build.gradle:48]: @jmatsu 
I see. My comment was off the maintain of PR.

**Review by satsukies — APPROVED** [review]
LGTM 👍

**Comment by jmatsu** [comment]
Thank you so much!

**Pull request merged**